tasks/db_deploy/bin/run_tests.sh

```diff
@@ -97,11 +98,11 @@ let return_code=$?
 check_rc $return_code "ERROR: Potential security or code issues found."
 
 
-## Check code third party libraries for CVE issues
-echo "INFO: Running security checks on third party libraries ..."
-safety check -r requirements.txt -r requirements-dev.txt
-let return_code=$?
-check_rc $return_code "ERROR: Potential security issues third party libraries."
+# ## Check code third party libraries for CVE issues
+# echo "INFO: Running security checks on third party libraries ..."
```

Comment:
Not sure why these are commented out. Isn't this security check still valid and should be done?

Reply:
oops forgot to remove the comments. 

Safety check was giving me some issue locally which is why I commented them in the first place. Looks good now